velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -198,6 +207,13 @@ void registerArrayFunctions(const std::string& prefix) {
   registerArrayHasDuplicatesFunctions<int128_t>(prefix);
   registerArrayHasDuplicatesFunctions<Varchar>(prefix);
 
+  registerArrayRemoveFunctions<int8_t>(prefix);
+  registerArrayRemoveFunctions<int16_t>(prefix);
+  registerArrayRemoveFunctions<int32_t>(prefix);
+  registerArrayRemoveFunctions<int64_t>(prefix);
+  registerArrayRemoveFunctions<int128_t>(prefix);
+  registerArrayRemoveFunctions<Varchar>(prefix);
```

Comment:
for Varchar you can optimize it by avoid copying strings see for example ArrayTrim function.

Reply:
@laithsakka , thanks for you comments.
optimized Varchar function by setNoCopy()

Done..